src/openrct2/world/map_generator/SimplexNoise.cpp

```diff
@@ -204,14 +204,15 @@ namespace OpenRCT2::World::MapGenerator
         resetSurfaces(settings);
 
         // Create the temporary height map and initialise
-        auto& mapSize = settings->mapSize;
-        auto heightMap = HeightMap(mapSize.x * 2, mapSize.y * 2);
+        const auto& mapSize = settings->mapSize;
+        const auto density = 2;
+        auto heightMap = HeightMap(mapSize.x * density, mapSize.y * density);
```

Comment:
This seems inelegant. Is there a reason this parameter is not passed in separately?

Reply:
I agree, it should probably be a parameter for HeightMap instead.